src/state/reducers/month-state/employee-info.reducer.ts

```diff
@@ -17,12 +17,29 @@ export function employeeInfoReducerF(name: string) {
     action: ScheduleActionModel
   ): WorkersInfoModel => {
     const data = action.payload?.employee_info;
-    if (!data) return state;
     switch (action.type) {
       case createActionName(name, ScheduleActionType.ADD_NEW):
-        return { ...data };
+        return { ...data! };
```

Comment:
Lepiej zostawić ten `if` co był poprzednio, ale przenieść go tutaj, ponieważ w reduserze, ty tak naprawdę nie możesz mieć pewności odnośnie tego, czy data nie jest undefindem. A jeśli zrobisz spread `undefined'a`, to polecą errory